xarray/core/variable.py

```diff
@@ -945,7 +944,7 @@ def load(self, **kwargs):
         --------
         dask.array.compute
         """
-        self._data = to_duck_array(self._data, **kwargs)
+        self._data = _maybe_wrap_data(to_duck_array(self._data, **kwargs))
```

Comment:
maybe we should just return the PandasExtensionArray wrapper class but I'm wary of exposing that to users

Reply:
OK i did this, it seems much neater at the expense of exposing the `PandasExtensionArray` wrapper class